documentation/MeetingMinutes/MOM-2023-09-20.md

```diff
@@ -0,0 +1,123 @@
+| N/A |  |  |
+| **New issues** |  |  |
+| [#59](https://github.com/camaraproject/IdentityAndConsentManagement/issues/59) | Orange | **documentation/CAMARA-API-access-and-user-consent.md diagram update purpose/scope**<br> Documentation/CAMARA-API-access-and-user-consent.md diagrams should be updated for POST/ authorize to use `scope` word instead of `purpose` word. A new PR will be created once we get agreement on issue #32. This new PR will correct the flow accordingly, and include the information about the agreed solution in the relevant doc sections. |
+| [#63](https://github.com/camaraproject/IdentityAndConsentManagement/issues/63) | Deutsche Telekom AG | **Usage of OAUTH2.0 client credentials in SimSwap API**<br> Moved to SIM Swap project as requested by Shilpa (DT). However, it was clarified that this issue actually stems from a discussion in the SIM Swap subproject where it was requested to move the discussion to this WG in order to clarify the agreed guidelines regarding the use of client credentials in CAMARA API access. The issue will be moved back to this WG and it is agreed to close it, as it is already clarified in the issue comments that client credentials do not apply to the SIM Swap API, nor to any other API processing personal user data. Dawid (DT) pointed out that in his opinion the introduction of CIBA into SimSwap looks like a "downgrade" of existing solutions that customers are using. But CIBA flow fits perfectly for the mass market,so it should be fine for CAMARA.<br>Issue will be moved back to this WG and CLOSED according to what is mentioned above.  |
+| **Open issues** |  |  | 
+| [#21](https://github.com/camaraproject/IdentityAndConsentManagement/issues/21) ([PR#33](https://github.com/camaraproject/IdentityAndConsentManagement/pull/33))| Deutsche Telekom AG | **Definition of scenarios for consent management**<br> No updates about this issue from last call. |
+| [#32](https://github.com/camaraproject/IdentityAndConsentManagement/issues/32) ([PR#64](https://github.com/camaraproject/IdentityAndConsentManagement/pull/64)) | Orange | **The concept of purpose**<br> There is still no agreement on the technical solution to declare a "purpose" when accessing the CAMARA API. This issue is already becoming a development bottleneck and needs to be resolved sooner rather than later. Telefonica proposes to set a deadline to try to find an agreement or an intermediate solution between the proposal of Telefonica and Orange. Hopefully we manage to do so, this is the desired way. But if there is no final agreement, it would be necessary to vote it according to CAMARA governance rules.<br>New PR #64 opened by Robert Ludovic (Orange) including a table to compare both solutions. Telefonica will complete it by Friday (22/09), then DT will review and add comments, maybe add new columns.<br>**Deadline will be the end of next week: 29/09. If no agreement is reached, the voting process would be organized for the week of 02/10.** |
```

Comment:
@jpengar What is meant by voting process here? Is this discussion + voting in TSC or simply open voting?

Reply:
@shilpa-padgaonkar Whatever is in line with CAMARA governance and TSC responsibilities. We can agree on the specific process to vote this properly. Whether it goes through TSC or not, I do not have a strong opinion on that. I don't know all the details of CAMARA governance, it's something that would have to be agreed if necessary. Hopefully we can come to an agreement next week to avoid a vote.

Reply:
@jpengar Thanks. Since commonalities and id-consent are not typical subprojects and as decisions made in these 2subprojects/wg impact all other subprojects, I just wanted to make sure that we do not intend to conduct any voting which is restricted only to "id-consent" members.  Hopefully this topic is resolved like you said by next week and we dont need to vote. If not, let's take it to TSC and trigger a discussion there to check if vote needs to be an open vote or if TSC can get it resolved without a vote.